### ISSUE TITLE
[Bug]: VSCode Extension doesn't list test cases in Test Explorer when configuration file is under sub folder of root of workspace.

### ISSUE THREAD

**Issue by iporsut** [microsoft/playwright#30869]
### Version

1.1.1 and 1.1.2

### Steps to reproduce

If you have and project structure like this

```
root-project
- api
- web
- e2e
  - playwright.config.ts
  - tests
    - test.spec.ts
```

### Expected behavior

I expect to see test cases in `e2e/tests` list in Test Explorer.

### Actual behavior

It doesn't show any test cases in Test Explorer

### Additional context

When I get back to install 1.0.22, it shows correctly.

### Environment

```shell
System:
    OS: macOS 14.4.1
    CPU: (8) arm64 Apple M1
    Memory: 42.88 MB / 16.00 GB
  Binaries:
    Node: 18.19.0 - ~/.asdf/installs/nodejs/18.19.0/bin/node
    Yarn: 1.22.21 - ~/.asdf/installs/nodejs/18.19.0/bin/yarn
    npm: 10.2.3 - ~/.asdf/plugins/nodejs/shims/npm
    bun: 1.0.33 - ~/.bun/bin/bun
  IDEs:
    VSCode: 1.89.1 - /usr/local/bin/code
  Languages:
    Bash: 5.2.26 - /opt/homebrew/bin/bash
  npmPackages:
    @playwright/test: ^1.44.0 => 1.44.0
```

**Comment by yury-s** [comment]
Related issue https://github.com/microsoft/playwright/issues/30819

**Comment by iporsut** [comment]
@yury-s Seem like it's already fixed in 1.1.3
